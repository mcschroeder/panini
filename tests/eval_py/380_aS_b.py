def f380(s: str):
  if s == "b":
    return
  else:
    i = 0
    while i < len(s):
      assert s[i] == "a"
      i += 1
