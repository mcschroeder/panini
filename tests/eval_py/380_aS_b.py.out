f380 : {s:𝕊 | s ∈ b|a*} → 𝟙  -- UNVERIFIED (timeout)